tpl/crawler/settings-general.tpl.php

```diff
@@ -28,6 +28,25 @@
 			</td>
 		</tr>
 
+		<tr>
+			<th>
+				<?php $id = Base::O_CRAWLER_SCHEDULE_TIME; ?>
+				<?php $this->title($id); ?>
+			</th>
+			<td>
+				<?php $this->build_input($id); ?>
+				<div class="litespeed-desc">
+					<?php echo __('Change the crawler running time.', 'litespeed-cache'); ?>
```

Comment:
What, specifically, does this refer to? Is it the time the crawler starts running? `running time` implies "how long it runs" so if this setting is for start times, change this to `Specify the time the crawler starts running.`

Reply:
Talked to Tynan about this.
Change to `Specify the crawler running times.`

Reply:
@lslisa Changed the text. Please confirm